integrations/using-integrations/computational-engines.md

```diff
@@ -0,0 +1,14 @@
+# Computational Engines
+
+{% hint style="info" %}
+Before starting, make sure you've added your Computational Engines as an integration to Aqueduct (see [adding-an-integration](../adding-an-integration/ "mention")).
```

Comment:
How do I do this? I don't see any documentation about this?

Reply:
![Screen Shot 2022-09-26 at 10 16 02 AM](https://user-images.githubusercontent.com/78303449/192339762-7f305e0e-7d98-4ce6-a8bd-6695746dc819.png)
I took a look at some other file, this is the a hint/suggestion column to ask users to connect integration on Aqueduct UI

Reply:
Thanks for clarifying, @Fanjia-Yan! Unfortunately, I don't think the Adding an Integration page has any information how to add compute integrations -- that's what we'll need to add more detail about here. Does that make sense?